synapse/tests/test_tools_json2mpk.py

```diff
@@ -0,0 +1,33 @@
+#from binascii import hexlify
+
+import synapse.tools.json2mpk as s_json2mpk
+
+from synapse.tests.common import *
+
+class Json2MpkTest(SynTest):
+
+    def test_tools_json2mpk(self):
+
+        with self.getTestDir() as ndir:
+
+            outp = self.getTestOutp()
+
+            path = os.path.join(ndir, 'woot.json')
+            newp = os.path.join(ndir, 'woot.mpk')
+            fake = os.path.join(ndir, 'fake.json')
+            html = os.path.join(ndir, 'woot.html')
+
+            self.false(os.path.isfile(path))
+            self.false(os.path.isfile(newp))
+
+            with open(path, 'wb') as fd:
+                fd.write(b'{"foo":10}\n{"bar":20}\n')
+
+            args = ['--rm', path, fake, html]
+
+            s_json2mpk.main(args, outp=outp)
+
+            with open(newp, 'rb') as fd:
+                self.eq(fd.read(), b'\x81\xa3foo\n\x81\xa3bar\x14')
```

Comment:
I would also add an assertion here that calling unpack on that data will result in the original JSON data.

Reply:
comparison of equality for them can be difficult because of dictionaries etc.  provided the bytes equal exactly this, i think the only thing we'd be testing would be the stability of the msgpack unpacker?